Tp rn rw (#239)
GovTechSG/techpass-user-docs: docs/sign-up-and-onboard-to-techpass.md

Users with a WOG account and who needs a TechPass account.

> ?>- Public officers using an **SE-GSIB** device need to submit a [service request](https://go.gov.sg/seed-techpass-support) to create their TechPass account.<br>- Alternatively, users can receive onboarding invitations for TechPass and SEED provisioning, which is optional, via the [**TechBiz portal**](https://portal.techbiz.suite.gov.sg) which is only accessible on a **non-SE GSIB** device. For more information, see [**TechBiz documentation**](https://docs.developer.tech.gov.sg/docs/techbiz-documentation/).

Reviewer: 2nd point is quite difficult to read. I presume the intention is to show the alternative route to sign up an account via TB.

if a po is at this page trying to self sign up, routing them to techbiz don't make sense.
I will imagine it is helpful for signing up on behalf (techbiz has the mechanism to track that). so i propose the following change.

```suggestion
?>- Public officers using an **SE-GSIB** device need to submit a [service request](https://go.gov.sg/seed-techpass-support) to create their TechPass account.<br>- Public officers can sign up on behalf of others (public officers/vendors) via [**TechBiz portal**](https://portal.techbiz.suite.gov.sg) for account and optional SEED provisioning. TechBiz portal is accessible only on a **non-SE GSIB** device. For more information, see [**TechBiz documentation**](https://docs.developer.tech.gov.sg/docs/techbiz-documentation/).
```

   ![sign-up-submit](assets/images/onboarding/po-non-se/latest-po-sign-up-non-se-gsib-1.png)

> 4. Click **Submit** to receive the onboarding invitation email(s). If you have also requested for SEED provisioning, you will receive a total of two onboarding invitation emails.

Reviewer: ```suggestion
4. Click **Submit** to receive the onboarding invitation email(s). If you have also requested for SEED provisioning, you will receive a total of two onboarding emails.
```

2. Read the **Terms of Use** and click **Accept**. You have successfully onboarded to TechPass.
3. If you had requested for SEED provisioning and is successfully provisioned to you, read the **MDM AUP Policy** and click **Accept**.

> >**Note**: You are only required to accept the terms and conditions when accessing any applications for the first time.

Reviewer: accepting of terms and conditions is compulsory to complete TP acct onboarding. 

```suggestion
>**Note**: You are required to accept the terms and conditions when accessing any applications for the first time.
```
